Counter/src/app/app.component.html

```diff
@@ -1,5 +1,8 @@
-<h1>Shiny Counter</h1>
-
-<p>A simple test.</p>
-<button routerLink="/detail">clicky</button>
+<div class="headerDiv">
+    <div class="imgDiv">
+        <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/98/International_Pok%C3%A9mon_logo.svg/1200px-International_Pok%C3%A9mon_logo.svg.png" alt="Pokemon logo" width="400px" height="auto">
```

Comment:
You'll generally want to avoid any inline styling like width and height that is directly on the element. Instead, put that styling in the CSS. Something like
```css
.headerDiv img {
    width: 400px;
}
```

Reply:
Fixed!